src/webgl/interaction.js

```diff
@@ -41,6 +52,17 @@ p5.prototype.orbitControl = function(sensitivityX, sensitivityY) {
   this._assert3d('orbitControl');
   p5._validateParameters('orbitControl', arguments);
 
+  var cam = this._renderer._curCamera;
+
+  // disable context menu for canvas element and add 'contextMenuDisabled'
+  // flag to p5 instance
```

Comment:
could we add an explanation here for why this is needed? e.g. right mouse button is used for which interactive component?

Reply:
Done!